hermes/hermes.quiver/hermes/quiver/model_repository.py

```diff
@@ -48,7 +49,12 @@ def __init__(
             self.fs = io.LocalFileSystem(root)
 
         if clean:
-            self.fs.remove("*")
+            # remove raises NoFilesFoundError if the directory is empty
+            try:
+                self.fs.remove("*")
+
```

Comment:
Purely pedantic, but I'm an advocate of [sparing use of blank lines](https://peps.python.org/pep-0008/#blank-lines), especially for a short block that's logically connected. I think a blank after `pass` makes sense to distinguish this block (what happens if we're supposed to clean the repo) from everything else, but this could probably be kept as a solid block. Again not super important and happy to be disagreed with, but something I thought I'd mention.

Reply:
Yes I agree, I added the line to trigger pre-commit and I thought black would've gotten rid of it. I guess not

Reply:
Ah yeah classic, yeah kinda surprised black doesn't have a hardline rule about this but whatever, I can live with it if black can